lib/mustache-parser.pegjs

```diff
@@ -21,6 +22,14 @@ newMustache = name:newMustacheStart m_* attrs:(m_bracketedAttrs / newMustacheAtt
   return ret;
 }
 
+m_blockParams = m_blockStart m_* "|" m_* params:blockParamNames+ "|" {
+  return params;
+}
+
+m_blockStart = "as"
+
+blockParamNames = newMustacheAttrValue
```

Comment:
This rule name would probably make more sense as a singular `blockParamName`. The result is going to be a single quotedString, valuePath or parenthetical, not an array.

In addition, the rule for a block param name should probably be more strict than this. I believe this will allow for something like, e.g. ```= my-component as |"abcdef"|`, which I do not think makes sense, but I'm inclined to let that fall through.

I think the comment @mixonic makes above would apply here: A possible simpler rule would be to simply match everything between the opening | and closing | as a single string. That would also fall prey to the issue I mention above, letting through nonsense strings inside the pipes.